src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -175,7 +175,11 @@ namespace mallocMC
              * existing chunk.
              * Each page can have 32x32 chunks. To maintain 32 chunks we need 32 bitmask on the page (each 32bit)
              */
-            static constexpr uint32 minChunkSize = (pagesize - 32u * sizeof(uint32)) / (32u * 32u);
+            static constexpr uint32 minChunkSizeUnaligned = (pagesize - 32u * sizeof(uint32)) / (32u * 32u);
+            // Set minChunkSize to largest multiple of minChunkSizeAlignment that is <= minChunkSizeUnaligned
+            static constexpr uint32 minChunkSizeAlignment = 16u;
+            static constexpr uint32 minChunkSize
```

Comment:
if you like to align you need to round up.

```
static constexpr uint32 minChunkSize
   = ((minChunkSizeUnaligned + minChunkSizeAlignment -1u)/ minChunkSizeAlignment) * minChunkSizeAlignment;
```

Reply:
I think that would be wrong as then the data might not fit the page. Probably easier to discuss on a VC